docs/app/Examples/elements/Button/States/ButtonLoadingExample.js

```diff
@@ -5,10 +5,10 @@ export default class ButtonLoadingExample extends Component {
   render() {
     return (
       <div>
-        <Button className='loading' />
-        <Button className='basic loading' />
-        <Button className='primary loading' />
-        <Button className='secondary loading' />
+        <Button loading />
+        <Button basic loading />
+        <Button loading className='primary' />
+        <Button loading className='secondary' />
```

Comment:
Regarding `primary` and `secondary`.  My first thought was to have these as boolean props, however, we could also do a value only prop:

```js
<Button primary />
<Button emphasis='primary' />
```

I like the first for brevity, however, there is also `positive` and `negative` which we could put under emphasis.  Thoughts?